doc/sphinx/Pacemaker_Explained/rules.rst

```diff
@@ -837,28 +1034,24 @@ Rules may be used to control a variety of options:
   ``template`` element)
 * :ref:`Operation properties <operation_properties>` (``meta_attributes``
```

Comment:
The idea is that "Resource options" covers only resource meta-attributes (which can be also set in op/op_defaults) and (newly renamed) "Operation options" refers to operation meta-attributes (which can only be set in op/op_defaults). I think this will be a little clearer with "as `meta_attributes` ..."

Reply:
> resource meta-attributes (which can be also set in op/op_defaults)

seriously??